tests/e2e/api.test.ts

```
@@ -1,59 +1,428 @@
+          status: 1,
+        } as TransactionReceipt;
+      }
+      throw new Error('Deposit not found for finalization in MockEndpointChainHandler');
+    }
+
+    async handleEndpointRequest(action: string, data: any): Promise<any> {
+      switch (action) {
+        case 'initialize_deposit':
+          if (!data || !data.id)
+            throw new Error('Missing deposit data for initialize_deposit via endpoint');
+          const initReceipt = await this.initializeDeposit(data as Deposit);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)